docs/reference/operations.md

```diff
@@ -319,6 +320,7 @@ post = await db.post.find_first(
             'endswith': 'must end with string',
             'in': ['find_string_1', 'find_string_2'],
             'mode': 'insensitive',
+            'search': 'full-text query',
```

Comment:
nit: might be nice to include an example of the special full-text syntax & link to either Prisma or psql / mysql docs for the syntax.